doc/source/installation.rst

```diff
@@ -4,7 +4,9 @@ Environment
 
 1. Install Gluster current stable release (https://www.gluster.org/download/)
 2. Install Etcd>=2.3.x && <3.x (https://github.com/coreos/etcd/releases/tag/v2.3.7)
-
+3. Install tendrl-node-agent using the document available at the url
+   https://github.com/Tendrl/node_agent/blob/master/doc/source/installation.rst
+   tendrl-node-agent is the package which is required on every storage node
```

Comment:
This is not required, the tendrl-node-agent is assumed to be present and it is the one which triggers installation of tendrl-gluster-integration